addons/uom/models/uom_uom.py

```diff
@@ -2,14 +2,25 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
 from odoo import api, fields, tools, models, _
-from odoo.exceptions import UserError
+from odoo.exceptions import UserError, ValidationError
 
 
 class UoMCategory(models.Model):
     _name = 'uom.category'
     _description = 'Product UoM Categories'
 
     name = fields.Char('Name', required=True, translate=True)
+    measure_type = fields.Selection([
+        ('unit', 'Units'),
+        ('weight', 'Weight'),
+        ('time', 'Time'),
+        ('length', 'Length / distance'),
```

Comment:
Why not just Length ?

Reply:
The complete name of the UoM category in odoo data is 'Length and distance'. I will change it.

Reply:
Is there any valuable difference between length and distance ?

Reply:
I don't think so ...